bitnami/harbor/templates/trivy/trivy-secret-envvars.yaml

```diff
@@ -14,6 +14,5 @@ metadata:
 type: Opaque
 data:
   SCANNER_TRIVY_GITHUB_TOKEN: {{ .Values.trivy.gitHubToken | default "" | b64enc | quote }}
-  SCANNER_STORE_REDIS_URL: {{ include "harbor.redisForTrivyAdapter" . | b64enc }}
```

Comment:
The upstream project seems to maintain the three env variables, which may be due to backwards compatibility with previous versions of the trivy container. What about keeping the three of them as well here to match this idea?

https://github.com/goharbor/harbor-helm/blob/f5767efaa35f835934ce060aa467b3a00af82837/templates/trivy/trivy-sts.yaml#L104

Reply:
Good idea, that seems safe(r) ;)